src/utils/network/session-core.js

```diff
@@ -398,269 +399,6 @@ export const createDocumentContainer = async (session) => {
   }
 };
 
-/*
-  User List Section
```

Comment:
Moved user list and user activity helpers into a separate file. Don't want `session-core` to become an unwieldy monolith

Reply:
I agree we need to refactor this out, but I think we should do this in a separate PR.